features/core-update.feature

```diff
@@ -32,6 +32,27 @@ Feature: Update WordPress core
       4.0
       """
 
+  @require-php-7.0
+  Scenario: Output in JSON format
```

Comment:
@ernilambar Can you add scenarios for `--format=table` and for `--format=csv` as well? We want to make sure those work as expected as well.